src/insert_pad.cpp

```diff
@@ -0,0 +1,95 @@
+#include <migraphx/insert_pad.hpp>
+#include <migraphx/program.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/op/convolution.hpp>
+#include <migraphx/op/im2col.hpp>
+#include <migraphx/op/pooling.hpp>
+#include <migraphx/op/pad.hpp>
+#include <migraphx/iterator_for.hpp>
+#include <migraphx/stringutils.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+
+void insert_pad::apply(module& p) const
```

Comment:
This should be `module& m`.

Reply:
I copied from elminate_pad.cpp, so I will change it there as well